src/helpers/fetchDateFromLocalStorage.js

```diff
@@ -0,0 +1,8 @@
+import { dataURL } from './dataURL';
+
+export const fetchDateFromLocalStorage = function () {
```

Comment:
Why did you choose this syntax for a function creation?

Reply:
I suggest you decided to trust your fate to AI again.

And as always it was wrong choice.